api/src/db/connectionConfig.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  host: process.env.DB_HOST || 'localhost',
+  user: process.env.DB_USER || 'postgres',
+  database: process.env.DB_NAME || 'spindle_dev',
+  password: process.env.DB_PASSWORD || '',
```

Comment:
The following is maybe not a concern...

How would your system react if you don't have these env variables in production?